components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/PostAuthnMissingClaimHandler.java

```diff
@@ -171,10 +179,22 @@ protected PostAuthnHandlerFlowStatus handlePostAuthenticationForMissingClaimsReq
                     }
                 }
 
+                List<LocalClaim> localClaims =
+                        getClaimMetadataManagementService().getLocalClaims(context.getTenantDomain());
+                Class<FrameworkUtils> claimDisplay = FrameworkUtils.class;
+                Object obj = claimDisplay.newInstance();
+                Method displayName = claimDisplay.
+                        getDeclaredMethod("getMissingClaimsDisplayNames", Map.class, List.class);
```

Comment:
Shall we get the `getMissingClaimsDisplayNames` method to this class as a private method.